domain/config/thresholds.py

```diff
@@ -0,0 +1,11 @@
+from typing import Protocol
+
+
+class Thresholds(Protocol):
```

Comment:
Separation of concerns; Thresholds can be imported and used without Config

Reply:
If Thresholds is put to config.py module, it can still be imported independently with `from domain.config.config import Thresholds` (or `from domain.config import Thresholds` with `domain.config.__init__.py` improvement). And doing one tiny class per file IMO breaks KISS principle.

Reply:
I see that you merged Thresholds and ThresholdOverride. It is sensible now.